tools/gyp/pylib/gyp/MSVSSettings.py

```diff
@@ -14,6 +14,10 @@
 MSBuild install directory, e.g. c:\Program Files (x86)\MSBuild
 """
 
+from __future__ import print_function
+
+from gyp import string_types
```

Comment:
This file appears to be missing.

Reply:
The __string_types__ variable is defined in the module's `__init__.py` file...
* https://github.com/nodejs/node/blob/master/tools/gyp/pylib/gyp/__init__.py#L19-L24

Reply:
Derp, can't believe I missed that.